packages/relay/tests/lib/eth/eth_call.spec.ts

```diff
@@ -200,97 +201,6 @@ describe('@ethCall Eth Call spec', async function () {
       process.env.ETH_CALL_DEFAULT_TO_CONSENSUS_NODE = initialEthCallConesneusFF;
     });
 
-    it('gas exceeds limit', async function () {
```

Comment:
Are we deleting them entirely ?

Reply:
@georgi-l95 
I think it makes sense to delete them:
* gas exceeds limit - we would no longer exceed the limit, since we cap it to the limit ourselves
* block 0, block 1, block earliest, block hash - these are also not relevant, since we have changed eth_call to work with historical block as well as block hashes